src/mds/objects.py

```diff
@@ -1,4 +1,4 @@
-from collections import Iterable
+from collections.abc import Iterable
```

Comment:
Fixes a [DeprecationWarning](https://github.com/uc-cdis/metadata-service/runs/1255470886) that appeared during test runs.